src/cmakeFindModules/OpenMVGConfig.cmake.in

```diff
@@ -28,141 +28,141 @@
 # Called if we failed to find Ceres or any of it's required dependencies,
 # unsets all public (designed to be used externally) variables and reports
 # error message at priority depending upon [REQUIRED/QUIET/<NONE>] argument.
-MACRO(OPENMVG_REPORT_NOT_FOUND REASON_MSG)
+macro(OPENMVG_REPORT_NOT_FOUND REASON_MSG)
   # FindPackage() only references Ceres_FOUND, and requires it to be
   # explicitly set FALSE to denote not found (not merely undefined).
-  SET(OPENMVG_FOUND FALSE)
-  UNSET(OPENMVG_INCLUDE_DIRS)
-  UNSET(OPENMVG_LIBRARIES)
+  set(OPENMVG_FOUND FALSE)
+  unset(OPENMVG_INCLUDE_DIRS)
+  unset(OPENMVG_LIBRARIES)
 
   # Reset the CMake module path to its state when this script was called.
-  SET(CMAKE_MODULE_PATH ${CALLERS_CMAKE_MODULE_PATH})
+  set(CMAKE_MODULE_PATH ${CALLERS_CMAKE_MODULE_PATH})
 
   # Note <package>_FIND_[REQUIRED/QUIETLY] variables defined by
   # FindPackage() use the camelcase library name, not uppercase.
-  IF (OPENMVG_FIND_QUIETLY)
-    MESSAGE(STATUS "Failed to find OPENMVG - " ${REASON_MSG} ${ARGN})
-  ELSE (OPENMVG_FIND_REQUIRED)
-    MESSAGE(FATAL_ERROR "Failed to find OPENMVG - " ${REASON_MSG} ${ARGN})
-  ELSE()
+  if(OPENMVG_FIND_QUIETLY)
+    message(STATUS "Failed to find OPENMVG - " ${REASON_MSG} ${ARGN})
+  else(OPENMVG_FIND_REQUIRED)
+    message(FATAL_ERROR "Failed to find OPENMVG - " ${REASON_MSG} ${ARGN})
+  else()
     # Neither QUIETLY nor REQUIRED, use SEND_ERROR which emits an error
     # that prevents generation, but continues configuration.
-    MESSAGE(SEND_ERROR "Failed to find OPENMVG - " ${REASON_MSG} ${ARGN})
-  ENDIF ()
-  RETURN()
-ENDMACRO(OPENMVG_REPORT_NOT_FOUND)
+    message(SEND_ERROR "Failed to find OPENMVG - " ${REASON_MSG} ${ARGN})
+  endif()
+  return()
+endmacro(OPENMVG_REPORT_NOT_FOUND)
 
 ## Public dependencies that needs to be propagated
 include(CMakeFindDependencyMacro)
 
-set(OpenMVG_USE_INTERNAL_CERES @OpenMVG_USE_INTERNAL_CERES@)
+set(OPENMVG_USE_INTERNAL_CERES @OPENMVG_USE_INTERNAL_CERES@)
 
-if(OpenMVG_USE_INTERNAL_CERES)
+if(OPENMVG_USE_INTERNAL_CERES)
```

Comment:
@fabiencastan @gregoire-dl with the trival these ifs are not correct anymore, as the options are now a string with `ON`, `OFF` and `AUTO`.
Maybe we can replace them with tests on `OPENMVG_HAVE_*` variables. To be check